docs/build-new/circuit-visualization.ipynb

```diff
@@ -0,0 +1,485 @@
+      "text/plain": [
+       "<Figure size 937.953x117.056 with 1 Axes>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Change the background color in mpl\n",
+    "\n",
+    "style = {\"backgroundcolor\": \"lightgreen\"}\n",
+    "circuit.draw(output=\"mpl\", style=style)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "ade9a653-3243-4ac9-bb0e-c8fb82f7a034",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAX0AAABBCAYAAAA9tnQzAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/H5lhTAAAACXBIWXMAAA9hAAAPYQGoP6dpAAADaklEQVR4nO3dP0ssdxQG4LMqXrBQgtjY2LmbIiL+QQKCKVKkSBWMTQqDTbASP0PafIEUQizSWFiGWIQEi1xEjZWyZRobQeINaOU4KSQXAl7vGFZ32PM83ezs/N7DFu8Mw+xuoyzLMgBIoa/bAwDwcpQ+QCJKHyARpQ+QiNIHSETpAySi9AESUfoAiSh9gESUPkAiSh8gEaUPkIjSB0hE6QMkovQBElH6AIkofYBElD5AIkofIBGlD5CI0gdIROkDJKL0ARIZ6PYAnVIWdxHlXWcXbfRFo7/aeVF+7vw6zCA/d35VPVH6ZXEXt19/E/Hm784uPDIcAz98/94PXX7u/DrMID93/lP0xu2d8q7zH3bE/ZpVztzyc+fXYQb5ufOfoDdKH4BKlD5AIkofIBGlD5BImtL/8+Y6Pnv9239e+/CXn+TLTzOD/Nz5/+raI5tXV1exsbERR0dH8erVq1hcXIzr6+vY2trq1kgAPa9rV/rLy8sxPT0dp6ensbe3F9vb2zE3N9etcQBS6MqV/v7+flxcXMTm5mZERIyNjcX4+HjMz88/a+4fb/6KT3//9Vkz5Muv8wzyc+dHdKn0Dw8PY2Fh4e325eVlnJ+fx9TU1DuPubm5iXa7/fDOooiPKuTOjHwQP3/8ydvtKvfTTk5OIvr7H3+T/Nz5dZhBftr8VqsVQ0NDFdLvdaX0x8bGYnd3N4qiiKIoYn19PZrNZgwODr7zmHa7HbOzsw/uG2g04ubzL59l1oWFhbgty0ffIz93fh1mkJ83//j4OGZmZiqv15XSX1lZiZ2dnWg2mzExMRGTk5MxOjr66DGtViuOj48f3lkUEd9+9wyTRhwcHFQ6y8tPnF+HGeSnzW+1Wk9ar1GWFS5jntnq6mosLS3F2tra/zq+vL2N2y++6vBU9wZ2f4zGwOPnRvm58+swg/zc+U9Ri+f0Dw8PPbkD8AJq8dPKZ2dn3R4BIIVaXOkD8DKUPkAiSh8gEaUPkIjSB0ikN0q/0RcxMtz5dUeG79eWL7/uM8jPnf8EtfhyVieUxV3H/0A4Gn2V/4Vefu78OswgP3d+5SV7pfQBeL/euL0DQCVKHyARpQ+QiNIHSETpAySi9AESUfoAiSh9gESUPkAiSh8gEaUPkIjSB0hE6QMkovQBElH6AIkofYBE/gGhob69nKGzewAAAABJRU5ErkJggg==",
+      "text/plain": [
+       "<Figure size 468.976x58.5278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Scale the mpl output to 1/2 the normal size\n",
+    "circuit.draw(output=\"mpl\", scale=0.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "495b560a-d835-448b-aa01-5025a7a0689e",
+   "metadata": {},
+   "source": [
+    "### Standalone circuit-drawing function\n",
+    "\n",
+    "If you have an application where you prefer to draw a circuit with a self-contained function instead of as a method of a circuit object, you can directly use the `circuit_drawer()` function, which is part of the public stable interface from `qiskit.visualization`. The function behaves identically to the `circuit.draw()` method, except that it takes in a circuit object as required argument."
```

Comment:
```suggestion
    "If you have an application where you prefer to draw a circuit with a self-contained function instead of as a method of a circuit object, you can directly use the `circuit_drawer()` function, which is part of the public stable interface from `qiskit.visualization`. The function behaves identically to the `circuit.draw()` method, except that it takes in a circuit object as a required argument."
```